### DISCUSSION TITLE
LGPL license in dependency?

### DISCUSSION THREAD

**Discussion by Zzzzzz-sw** [florisboard/florisboard#1907] [General]
First I would like to thank the devs for this great app. It is hard to find high-quality open-source keyboards these days and thank you for your contribution!

Since 0.4.0 is next which wants to re-enable word suggestions, I would like to draw attention to this issue:
https://twitter.com/GrapheneOS/status/1519143831832305664
It seems that the current spellchecker has a restrictive license that prevents others using it having permissive license. Is there any consideration to move away from the library, or have a separate version that excludes it?

**Comment by patrickgold** [comment]
Thanks a lot for drawing my attention to this!

Please see https://github.com/GrapheneOS/os-issue-tracker/issues/579#issuecomment-1156507221 , I am very open to get rid of Nuspell and integrate this into the same system as the upcoming inline autocorrect if it means that it can be included in GrapheneOS.

**Reply by Zzzzzz-sw** [reply]
Thanks for the quick response! Hope you can work things out in the end <3
